core/src/test/resources/sql/misc.iq

```diff
@@ -473,7 +473,7 @@ EnumerableCalc(expr#0..7=[{inputs}], expr#8=[IS NULL($t5)], expr#9=[IS NULL($t7)
           EnumerableCalc(expr#0..3=[{inputs}], expr#4=[true], deptno=[$t0], $f0=[$t4])
             EnumerableTableScan(table=[[hr, depts]])
     EnumerableAggregate(group=[{0}], agg#0=[MIN($1)])
-      EnumerableCalc(expr#0..3=[{inputs}], expr#4=[90], expr#5=[+($t0, $t4)], expr#6=[true], $f4=[$t5], $f0=[$t6])
+      EnumerableCalc(expr#0..3=[{inputs}], expr#4=[90], expr#5=[+($t0, $t4)], expr#6=[true], $f4=[$t5], $f0=[$t6], $condition=[$t6])
         EnumerableTableScan(table=[[hr, depts]])
 !plan
```

Comment:
Is this plan change expected?

Reply:
Yes, since the total cost is the same.

Reply:
I think optimizer latency can be benefited a lot from this patch. Any experiment?

Reply:
Not yet specific experiment. But the slow tests in the patch is 23m. Comparing 32m in master.